### PR TITLE
Codegen docs

### PR DESCRIPTION
**Pull request by paulbakker**

### CODE REVIEW

File: docs/generating-code-from-schema.md [hunk 1]
```diff
@@ -8,19 +8,22 @@ The plugin generates the following:
...
+	id "com.netflix.dgs.codegen" version "4.2.0"
 }
 ```
 
 Alternatively, you can set up classpath dependencies in your buildscript:
 ```groovy
 buildscript {
    dependencies{
-      classpath 'com.netflix.graphql.dgs.codegen:graphql-dgs-codegen-gradle:latest.release'
+      classpath 'com.netflix.graphql.dgs.codegen:graphql-dgs-codegen-gradle:4.2.0'
```
Comment (berngp) [inline docs/generating-code-from-schema.md:26]: Add a disclaimer on using the latest version? the version here is used as example.
We could also use `4.2.+`?

**Review by berngp — APPROVED** [review]

File: docs/generating-code-from-schema.md (hunk 1 above)
Comment (paulbakker) [reply docs/generating-code-from-schema.md:26]: This doesn't work for plugins - only for normal dependencies

**Review by srinivasankavitha — APPROVED** [review]

File: docs/generating-code-from-schema.md (hunk 1 above)
Comment (berngp) [reply docs/generating-code-from-schema.md:26]: Sorry, thanks for the clarification.

Comment (paulbakker) [reply docs/generating-code-from-schema.md:26]: I want to look into automating updating the versions in the docs each time we release, maybe based on a github action.
That's a todo though

Comment (paulbakker) [reply docs/generating-code-from-schema.md:26]: I'll make it 4.+, I think that's the best we can currently do?

**Pull request merged**